Fhv/sdk docs (#121)
encord-team/encord-client-python: docs/source/tutorials.rst

*********
Tutorials
*********

> We have separated the tutorials into those that relate to *datasets*, those that relate to *projects* and those that give you end-to-end examples of common scenarios.

Reviewer: ```suggestion
The tutorials are separated into those that relate to *datasets*, those that relate to *projects* and those that give you end-to-end examples of common scenarios.
```
